sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslAggregationTest.java

```diff
@@ -1010,4 +1012,39 @@ public void testCountIfFunction() throws Exception {
     PAssert.that(result).containsInAnyOrder(rowResult);
     pipeline.run().waitUntilFinish();
   }
+
+  @Test
+  public void testCovarPopFunction() throws Exception {
```

Comment:
We already have a test for covar_pop in https://github.com/apache/beam/blob/master/sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslAggregationCovarianceTest.java.